rubin_sim/skybrightness/interpComponents.py

```diff
@@ -562,15 +567,19 @@ def interpMag(
         )[0]
 
         for i, filterName in enumerate(filterNames):
-            result[good, i] = twilightFunc(
-                interpPoints[good],
-                *self.lsstEquations[self.filterNameDict[filterName], :].tolist()
-            )
+            out_of_range = np.where(interpPoints["sunAlt"] > np.max(limits))[0]
+            if np.size(out_of_range) > 0:
+                result[:, i] = np.nan
+            else:
+                result[good, i] = twilightFunc(
+                    interpPoints[good],
+                    *self.lsstEquations[self.filterNameDict[filterName], :].tolist()
+                )
 
         return {"spec": result, "wave": self.lsstEffWave}
 
     def interpSpec(
-        self, interpPoints, maxAM=3.0, limits=[np.radians(-5.0), np.radians(-20.0)]
+        self, interpPoints, maxAM=3.0, limits=[np.radians(15.0), np.radians(-20.0)]
```

Comment:
It would be better for the default of "limits" to be a tuple (which is immutable) rather than a list (which is mutable, so can lead to unexpected consequences).